### PR TITLE
Add `sendPoll` endpoint

### PR DESCRIPTION
**Pull request by pavelgnom**
This PR covers `sendPoll` and `stopPoll` endpoints.

### CODE REVIEW

**Comment by tjhorner** [comment]
@pavelgnom tag me when this is ready for review, will take a look :)

**Comment by pavelgnom** [comment]
@tjhorner can you please take a look?

File: bot_test.go [hunk 1]
```diff
@@ -425,7 +426,30 @@ func TestSendWithDiceWithEmoji(t *testing.T) {
...
+	}
+}
+
+func TestSendWithStopPoll(t *testing.T) {
+	bot, _ := getBot(t)
 
+	stopPoll := tgbotapi.NewStopPoll(ChatID, 59401)
+	_, err := bot.Send(stopPoll)
+
+	if err != nil && !strings.Contains(err.Error(), "poll has already been closed") {
```
Comment (pavelgnom) [inline bot_test.go:449]: @tjhorner do you have any thoughts how could we get rid of it?

File: configs.go [hunk 2]
```diff
@@ -217,6 +223,67 @@ func (config MessageConfig) method() string {
 	return "sendMessage"
 }
 
+// PollConfig contains information about a SendPoll request.
+type PollConfig struct {
+	BaseChat
+	Question              string
+	Options               []string
+	Anonymous             bool
```
Comment (tjhorner) [inline configs.go:231]: Let's change this to `IsAnonymous` to keep it consistent with the other fields prefixed with `is_` in the API.

File: helpers.go [hunk 3]
```diff
@@ -18,6 +18,35 @@ func NewMessage(chatID int64, text string) MessageConfig {
...
 
+// NewPoll creates a new Poll.
+//
+// chatID is where to send it, question is the poll title,
+// options – list of options.
+func NewPoll(chatID int64, question string, options []string) PollConfig {
+	return PollConfig{
+		BaseChat: BaseChat{
+			ChatID:           chatID,
+			ReplyToMessageID: 0,
```
Comment (tjhorner) [inline helpers.go:29]: Wouldn't this be `0` by default?

**Review by tjhorner — CHANGES_REQUESTED** [review]
LGTM, just a name change and question.

**Review by tjhorner — APPROVED** [review]

**Comment by tjhorner** [comment]
@Syfaro mind taking one last look?

**Comment by pavelgnom** [comment]
@Syfaro @tjhorner do we have any estimate on when that could be merged? Thanks.

**Comment by pavelgnom** [comment]
I've put some minor improvements there and rebased it with the latest master.
@tjhorner @Syfaro can you please review it once again?

**Comment by Syfaro** [comment]
I'm sorry it took so long to review this, I really do appreciate the contribution. 

However the `sendPoll` method was implemented in the develop branch a while ago, and that just got merged in so this is no longer needed.

**Pull request closed without merging**